lib/plugin.rb

```diff
@@ -59,6 +59,13 @@ def request_params
       params["plugin[category]"] = @manifest["type"]
       params["plugin[external_identifier]"] = @identifier
       params["plugin[whitelisted_account_ids][]"] = @manifest["whitelisted_account_ids"]
+      params["plugin[guide]"] = @plugin_guide || ""
```

Comment:
why we need this? what will happen if you send `nil`?

Reply:
I admit I didn't test it as both javascript & ruby will treat empty string as a falsy value - so I thought this was safer than allowing the value to be nil

Reply:
not sure I understand why you keep it. When posting nil value on request the values will be ignored. Why do we want empty strings in the db?